libs/SalesforceSDKCore/SalesforceSDKCore/Classes/PushNotification/PushNotificationManager.swift

```diff
@@ -317,64 +305,14 @@ public class PushNotificationManager: NSObject {
         return unregisterSalesforceNotifications(for: user, completionBlock: completionBlock)
     }
     
-    /// Unregisters the device from Salesforce push notifications for a specific user.
-    ///
-    /// - Parameters:
-    ///   - user: The user account to unregister.
-    ///   - completionBlock: A block executed when unregistration is complete.
-    /// - Returns: `true` if unregistration started successfully, otherwise `false`.
-    @discardableResult
-    @objc(unregisterSalesforceNotificationsWithCompletionBlock:completionBlock:)
-    public func unregisterSalesforceNotifications(for user: UserAccount,
```

Comment:
Making this private looks like it would be a breaking change from 13.0.x

https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/096b4e515c4f61487c03ed0279363eb57c09c092/libs/SalesforceSDKCore/SalesforceSDKCore/Classes/PushNotification/SFPushNotificationManager.h#L100